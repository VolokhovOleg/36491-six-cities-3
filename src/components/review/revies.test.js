import Revies from './revies';

const review = {
  text: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
  userRate: 4,
  userName: `Max`,
  date: `December 24, 2018`,
  id: `1`,
};

const {text, userRate, userName, date, id} = review;

it(`<Revies /> snapshot`, () => {
  const tree = renderer
    .create(<Revies
      key={id}
      text={text}
      userRate={userRate}
      userName={userName}
      date={date}
      id={id}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
