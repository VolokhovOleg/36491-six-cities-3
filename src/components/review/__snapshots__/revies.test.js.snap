// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Revies /> snapshot 1`] = `
<li
  className="reviews__item"
  id="1"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height={54}
        src="img/avatar-max.jpg"
        width={54}
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Max
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.
    </p>
    <time
      className="reviews__time"
      dateTime="2018-12-24"
    >
      December 2018
    </time>
  </div>
</li>
`;
